### PR TITLE
[WIP] Augur core docker fix 20190503

### PR DESCRIPTION
**Pull request by petong**
* deletes venv dir if it exists, which would cause an error building 
* removes docker pull from run.sh so that if the image exists locally already use that, if not, fetch from docker hub
* add debug info to pop-geth image, showing last block after deploy
* tags augur-core docker image with contracts md5
* gets rid of python2 in docker image
* removes old node from augur.js matrix

### CODE REVIEW

**Review by bthaile — APPROVED** [review]
lgtm, lets do this

File: packages/augur-tools/docker-builder/run.sh [hunk 1]
```diff
@@ -4,6 +4,6 @@ IMAGE=$1
 TAG=$(node ./scripts/get-contract-hashes.js)
 ./scripts/copy-docker-files.sh $IMAGE:$TAG
 echo "Running Geth with contracts deployed from ${IMAGE}:${TAG}"
-docker pull $IMAGE:$TAG
+#docker pull $IMAGE:$TAG
```
Comment (justinbarry) [inline packages/augur-tools/docker-builder/run.sh:7]: Is this debug code?

Comment (petong) [reply packages/augur-tools/docker-builder/run.sh:7]: no, we don't need that, running the image will pull from docker hub, but if you have it locally the pull will overwrite it. We should just delete the whole line

Comment (pgebheim) [reply packages/augur-tools/docker-builder/run.sh:7]: We actually just decided NOT to do that.

@petong lets just remove?

```suggestion
```

Comment (pgebheim) [reply packages/augur-tools/docker-builder/run.sh:7]:
```suggestion
```

Shall we remove it @petong

**Comment by petong** [comment]
I made a new PR with the CI optimizations, and will include the python2 -> python3 changes separately

**Pull request closed without merging**